packages/charts/chart-web-components/scripts/clean.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-undef */
```

Comment:
can we add notes to all of these files that those are copy paste from web-components project in order to replace this in future with proper nx inferred project setups we have in place for react components v9 ?  (issue which link would be present in these files as comment would be ideal ). ty

Reply:
Added comments and link to tracking issue in the scripts and config files.